xdsl/ir/core.py

```diff
@@ -594,11 +593,9 @@ def __init_subclass__(cls) -> None:
 
         cls.enum_type = enum_type
 
-    @final
```

Comment:
why does the base not use in_angle_brackets?

Reply:
Because not all enum attributes use this syntax; in fact, most of them don't:
`omp<schedule static>`: the name part is `omp<schedule` ... `>`, the parameter part is just ` static`

Reply:
Also, after #2256, I'm thinking this dominating case of having brackets could be made a baseclass handled in our printer? :thinking: